projects/adrv9009/src/devices/adi_hal/parameters.h

```diff
@@ -137,6 +137,10 @@
 #define TX_DMA_BASEADDR			XPAR_AXI_ADRV9009_TX_DMA_BASEADDR
 
 #define DDR_MEM_BASEADDR		XPAR_DDR_MEM_BASEADDR
+
+#define UART_DEVICE_ID			XPAR_XUARTPS_0_DEVICE_ID
+#define UART_IRQ_ID			XPAR_XUARTPS_1_INTR
```

Comment:
Is this the correct interrupt ID for UART device 0? I was expecting it to be `XPAR_XUARTPS_0_INTR`.